installer/pkg/config/v1/config.go

```diff
@@ -5,21 +5,55 @@
 package config
 
 import (
+	"github.com/gitpod-io/gitpod/installer/pkg/config"
 	"github.com/gitpod-io/gitpod/ws-daemon/pkg/resources"
+
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/utils/pointer"
 )
 
+func init() {
+	config.AddVersion("v1", v{})
+}
+
+type v struct{}
```

Comment:
maybe `version`? makes more readable. especially when you write `func (v v) Factory() interface{} { return &Config{} }
`

Reply:
:+1: